index.html

```diff
@@ -0,0 +1,162 @@
+             <img src="https://www.iconsdb.com/icons/preview/orange/truck-2-xxl.png" alt="truck">
+          </div>
+         <div class="container-text">
+             <h1>Fast Shipping</h1>
+             <p>We make sure you recieve your trombone as soon as we have finished making it.
+                  We also provide free returns if you are not satisfied.</p>
+             
+         </div>
+       
+          </div>
+          <div id="containerBox">
+              <div class="container-icon">
+            <i class="fa fa-3x fa-battery-full"></i>
+            <img src="https://www.iconsdb.com/icons/preview/orange/full-battery-xxl.png" alt="battery">
+              </div>
+              <div class="container-text">
+                  <h1>Quality Assurance</h1>
+                  <p>For every purchase you make, we will ensure there ar
+                      e no damages or faults and we will check and test the pitch of your instrument.</p>
+            
+              </div>
+            
+               </div>
+  
+    </section>
+  
+  
+    <section id="HowItWorks" class="video">
+  
+      <iframe width="560" height="315" src="https://www.youtube.com/embed/y8Yv4pnO7qc?start=44" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; 
+      gyroscope; picture-in-picture" allowfullscreen></iframe>
+   </section>
+  
+   <section id="Pricing">
+  
+      <div class="box-wrapper">
+          <div class="box">
+              <h1>TENOR TROMBONE</h1>
+              <p><strong>$600</strong></p>
+              <p>Lorem Ipsum</p>
+                 <p>Lorem Ipsum</p>
+                 <p>Lorem Ipsum Dorem</p>
+                 <p>Lorem Ipsum</p>
+                 <a href="" class="btn">SELECT<a>
+  
+          </div>
+  
+  
+  
+          <div class="box">
+              <h1>BASS TROMBONE</h1>
+              <p><strong>$900</strong></p>
+              <p>Lorem Ipsum</p>
+                 <p>Lorem Ipsum</p>
+                 <p>Lorem Ipsum Dorem</p>
+                 <p>Lorem Ipsum</p>
+                 <a href="" class="btn">SELECT<a>
+  
+          </div>
+  
+  
+  
+          <div class="box">
+              <h1>VALVE TROMBONE</h1>
+              <p><strong>$1200</strong></p>
+              <p>Plays similar to a Trumpet</p>
+              <p>Lorem Ipsum</p>
+              <p>Lorem Ipsum Dorem</p>
+              <p>Lorem Ipsum</p>
+              <a href="" class="btn">SELECT<a>
+              
+          </div>
+  
+      </div>
+   </section>
+  
+   <footer>
+       <div class="footer">
+           <ul>
+               <li><a href="">Privacy</a></li>
+               <li><a href="">Terms</a></li>
+               <li><a href="">Contact</a></li>
+           </ul>
+  
+       </div>
+       <div class="footer-copyright">
+           <p>Copyright 2016, Original Trombones</p>
+       </div>
+   </footer>
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+    </div>
+  </body>
+</html>
```

Comment:
It's good to explain why you're asking for the changes but you should first appreciate the person's efforts, also if you have any relevant resources or examples to share, include them in the comment. This can help the person to better understand your suggestion and how to implement it.